File: condicoes/1044.py
#1044 MÚLTIPLOS
valores = input().split(" ")
valorA = int(valores[0])
valorB = int(valores[1])

if valorA%valorB == 0 or valorB%valorA == 0:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")

File: javascript/1044.js
/*Múltiplos*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split(" ");

let A = parseInt(lines.shift());
let B = parseInt(lines.shift());

if(A % B == 0 || B % A == 0){
  console.log("Sao Multiplos");
}else{
  console.log("Nao sao Multiplos");
}